dialogue/TownNPCs/Mocha/InitialInteractions/Banana/Banana_Mocha_First1.json

```diff
@@ -13,6 +13,10 @@
       "¡Bievenidos a la posada de Cornucopia!",
       "Anda, ¿no te he visto antes?\n@CharacterName, ¿verdad?\nTu también estuviste persiguiendo a <c=violastro>ese villano</c> ¿no?"
     ],
+    "pl": [
+      "Witam pana do własnej karczmy Cornucopii!",
+      "O, skąd pana znam?\n@CharacterName, nie?\nCzy pan nie też gonił <c=violastro>tamtego łotra</c>?"
```

Comment:
A small thing, but I think it will be better to put "też" before "nie"

Reply:
I was having trouble deciding this, since i thought that "też nie" would make it sound as if he didn't chase the villain.

Reply:
Hmm, you're kinda right, but I think I would understand both of these versions the same way, so I think it's pretty much up to you then